services/mapper.go

```diff
@@ -130,7 +130,9 @@ func TransferOps(tx *evmClient.LoadedTransaction, startIndex int) []*RosettaType
 	return ops
 }
 
-func FeeOps(tx *evmClient.LoadedTransaction) []*RosettaTypes.Operation {
+func FeeOps(tx *evmClient.LoadedTransaction, args ...interface{}) []*RosettaTypes.Operation {
```

Comment:
this function argument change can potentially break all upstream Rosetta (e.g. Base, Arbitrum, BSC). regression tests for different chains are needed for this change